### PR TITLE
usermanual: document double-click on a copy history dialog.

### PR DESCRIPTION
**Pull request by TurboGit**

### CODE REVIEW

File: doc/usermanual/lighttable/panels/history.xml [hunk 1]
```diff
@@ -47,6 +47,8 @@
         Copy the history stack of the selected image. You will be prompted for which items are
         to be include. If more than one image is selected, the history stack is taken from the
         image that has been selected first.
+
+        A double-click on an item copies this one only and close the dialog.
```
Comment (Balise42) [inline doc/usermanual/lighttable/panels/history.xml]: I'd go for "copies that item" (but then I'm somewhat fuzzy on this vs that, so....)

File: doc/usermanual/lighttable/panels/history.xml [hunk 2]
```diff
@@ -47,6 +47,8 @@
         Copy the history stack of the selected image. You will be prompted for which items are
         to be include. If more than one image is selected, the history stack is taken from the
```
Comment (Balise42) [inline doc/usermanual/lighttable/panels/history.xml]: to be included

Comment (brucewilliamsphotography) [inline doc/usermanual/lighttable/panels/history.xml]: Copy the history stack of the selected image. You will be prompted for which items are to be included.

File: doc/usermanual/lighttable/panels/history.xml (hunk 1 above)
Comment (brucewilliamsphotography) [reply doc/usermanual/lighttable/panels/history.xml]: No, Balise42, you're spot on there. It should be "copies that item".

**Comment by TurboGit** [comment]
Thanks!! That is merged now.

**Pull request closed without merging**